### ISSUE TITLE
Does the load() method also load the return values of fit_transform()?

### ISSUE THREAD

**Issue by DerSteph** [MaartenGr/BERTopic#544]
Hello,
I use BERTopic for my coursework and I am very satisfied compared to older methods like LDA and LSI. Since I am doing a trend analysis on a larger dataset from my university, I wanted to know if I can also load the return values of fit_transform() with load().

**Comment by MaartenGr** [comment]
@DerSteph Fortunately, the labels, and probabilities of the most recent fit are saved in the HDBSCAN model from which they can be extracted. To extract them, you can try the following:

```python
probs = topic_model.hdbscan_model.probabilities_
topics = topic_model._map_predictions(topic_model.hdbscan_model.labels_)
```

Do note that these are the flat probabilities.

**Comment by DerSteph** [comment]
Thank you, that helped me a lot!
